views/applicationForm.html

```diff
@@ -47,10 +47,51 @@ <h1>429 Manager Application Form</h1>
         <option value="D">D</option>
         <option value="F">F</option>
       </select>
-  
-      <label for="fileAttachment">File Attachment:</label>
+      <label for="time-preference" class="table-label">Time Preference:</label>
+      <table class="time-preference">
+ <table class="time-preference">
+  <thead>
+    <tr>
+      <th></th>
+      <th>1st</th>
+      <th>2nd</th>
+      <th>3rd</th>
+      <th>4th</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Section 01: T/TH 2:30-3:45 PM <br>Discussion: W 10:10-11:00 AM</td>
+      <td><input type="radio" name="section1_time" value="1"></td>

```

Comment:
The little borders around the input of type radio kinda bugs me but I can't really find a way to fix it.